pymysql/connections.py

```diff
@@ -1100,6 +1131,11 @@ def _read_ok_packet(self, first_packet):
         self.message = ok_packet.message
         self.has_next = ok_packet.has_next
 
+    def _read_load_local_packet(self, first_packet):
+        load_packet = LoadLocalPacketWrapper(first_packet)
+        local_packet = LoadLocalFile(load_packet.filename, self.connection)
+        local_packet.send_data()
+
```

Comment:
I'm not sure what you are getting at here.. This is not an ok_packet, it has a different format than an ok packet and needs to be handled differently...